spec/library_spec.rb

```diff
@@ -0,0 +1,51 @@
+require './lib/library.rb'
+require './lib/person.rb'
+describe Library do
+  person = Person.new("Carissa")
+  subject { described_class.new()}
+  it 'should load books from a file' do
+    expect(subject.books.length).to eq 5
+  end
+
+  it 'should display title and author of available books' do
+    first_item = subject.books[0]
+    expect(first_item[:item][:title]).to be_truthy
```

Comment:
Why not check that the title matches the name of the first book in your collection? in Ruby anything that's not `nil` or `false` will always `be_truthy` so even if the title has some random value in it, this test will not catch it